config/webpack/loaders/devSiteMarked.js

```diff
@@ -51,19 +51,29 @@ const loader = async function loader(markdown) {
   const code = [
     'import React from \'react\';',
     'import classNames from \'classnames/bind\';',
+    'import { ThemeContext } from \'terra-application/lib/theme\'',
     'import styles from \'terra-dev-site/lib/marked/marked.module.scss\';',
     '',
     'const cx = classNames.bind(styles);',
     '',
-    `export default () => (
-      <div
-        dir="ltr"
-        className={cx('marked')}
-        dangerouslySetInnerHTML={{
-          __html: '${html}',
-        }}
-      />
-    );`,
+    `export default () => {
+      const theme = React.useContext(ThemeContext);
+      const markedClassNames = classNames(
+       cx([
+         'marked',
+         theme.className
+       ])
+      );
```

Comment:
The outer classNames can be removed, it's only necessary when combining a non-locally scoped class name:
```suggestion
      const markedClassNames = cx('marked', theme.className);
```